fearless/Modules/Staking/StakingMain/View/StoriesPreviewCollectionItem.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+import SoraUI
+
+class StoriesPreviewCollectionItem: UICollectionViewCell {
+
+    @IBOutlet private var iconLabel: UILabel!
+    @IBOutlet private var captionLabel: UILabel!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        layer.masksToBounds = true
+        layer.cornerRadius = 4.0
+        layer.borderWidth = 1.0
+        layer.borderColor = UIColor.init(white: 1.0, alpha: 0.5).cgColor
```

Comment:
Did you consider to use RoundedView as background from SoraUI?

Reply:
Looks like it doesn't support transparency

Reply:
Oh, it does. Done